mkdocs.yml

```diff
@@ -37,9 +37,10 @@ nav:
     - Default Environment: default-environment.md
   - Examples: examples.md
   - Upgrade:
-    - 2.0.0: migration/2.0.0-migration.md
+    - 2.0: migration/2.0-migration.md
   - Development:
     - Local development: development-local.md
+    - Versioning: versioning.md
     - Custom Images: custom-image.md
```

Comment:
For the category, the page is under `Development`, but I'm open to suggestions if anyone has a better idea where to place the `Versioning` page.

Reply:
IMO, I would put it under configuration.  Development to me is more about customizing an image.

Reply:
Just a thought.

Reply:
The "Configuration" section would work better indeed.
Updated in https://github.com/bitovi/bitops/pull/278/commits/50e358b2ebba2f7429d4df252dce8d43aa1e0bef, thanks!